src/plugin.h

```diff
@@ -0,0 +1,78 @@
+#pragma once
+
+namespace kaleidoscope {
+
+class Kaleidoscope_;
+
+struct EventKey {
```

Comment:
The parameters of the old eventHandlerHook have been grouped in struct `EventKey`. This enables adding further parameters in the future without breaking the interface.